commcare_connect/opportunity/tables.py

```diff
@@ -876,6 +876,8 @@ class Meta:
     def __init__(self, *args, **kwargs):
         organization = kwargs.pop("organization", None)
         super().__init__(*args, **kwargs)
+        self.use_view_url = False
```

Comment:
What does this do?

Reply:
This was for using the table url for header sorting when the table is loaded.